src/main/scala/cromwell/Main.scala

```diff
@@ -80,7 +80,7 @@ class Main private[cromwell](enableTermination: Boolean, managerSystem: () => Wo
   lazy val Log = LoggerFactory.getLogger("cromwell")
   Monitor.start()
 
-  def this() = this(enableTermination = true, managerSystem = () => CromwellServer)
+  def this() = this(enableTermination = true, managerSystem = () => new WorkflowManagerSystem {})
 
   // CromwellServer still doesn't clean up... so => Any
   def runAction(args: Seq[String]): Any = {
```

Comment:
`parseInputs` is no longer used in this file.  Is that a result of your previous de-redundification?